plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/config/CPSConfiguration/help-showSandbox.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>Controls whether the "Use Groovy Sandbox" is shown in pipeline jobs configuration to users without Overall/Administer permission.</p>
+    <p>Note that this is not a security configuration, as it does not prevent users to configure and run
+        pipelines in some other ways without using the sandbox.</p>
```

Comment:
Enhancements under discusssion:
* Throw `AbortException` if an attempt is made to run a non-sandboxed pipeline.
* Throw something from the `@DataBoundConstructor` if `!sandbox`.

The first option may not work though, because we allow an admin to define a non-sandboxed pipeline (maybe some special admin/diagnostic job), and we would not know when the build starts who edited it.